tools/nbs/extract_images.py

```diff
@@ -19,6 +25,82 @@ def register(command):
     command.add_argument("--output", "-o", nargs="?", default=os.getcwd(), help="The folder to extract the images into")
 
 
+# Gets the image and debayers if needed, then saves the image
+def process_image(packet, output):
```

Comment:
```suggestion
def process_save_image(packet, output):
```
I think this describes the function better.